README.md

```diff
@@ -1,12 +1,15 @@
 # app-http-logger
+
```

Comment:
oof... seems my autoformatter had its way with this file... apologies... will mark the relevant section with a comment